test/tap/tests/test_pgsql_replication_lag/delete_primary_vintf_throttle.sh

```
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+rsubnet=$(docker exec --user root pgsql-repl-pg_replica-1 sh -c \
+	'getent hosts pg_primary_int | cut -d" " -f1')
+vintf=$(docker exec --user root pgsql-repl-pg_primary-1 sh -c \
+	"echo $rsubnet | xargs ip route show to match | grep -E "$rsubnet" | cut -d' ' -f3")
```

Shellcheck (0.11.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)